port/zos390/omrsimap.h

```diff
@@ -48,7 +48,7 @@
  * The RSM Internal Table (RIT) reached from the `ritp' address field in PVT. The
  * field of interest here is the total amount of online storage.
  */
-typedef _Packed struct J9RIT {
+typedef struct __attribute__((packed)) J9RIT {
```

Comment:
Again, do xlc and OpenXL attach the same meaning to this?

Reply:
They do have the same meaning as per discussing internally with Sean from the cpp team. I did inquire if we have any published/public documentation for this, but I dont think they have a dedicated page regarding this particular suggestion/tweak.
However digging a bit and searching for it, there is a small mention of it on this page: https://www.ibm.com/docs/en/open-xl-c-cpp-zos/2.1?topic=guide-binary-compatibility

To quote part of our discussions: 

> The problem with _Packed is that it is C only and the usual change to attribute((packed)) for C++ doesn't have the same semantics.
```c
typedef struct S _Packed {
   char c;
   int i;
 } S;
 S s1;
 struct S s2;
 ```
> In this example s1 and s2 have different sizes and types.  The _Packed keyword applies to the declarator (eg. typedef name S) on the struct definition.  The _Packed keyword creates a hidden duplicate version of the struct.
> Avoid the keyword even in XL.